release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -964,6 +964,19 @@ See xref:../release_notes/versioning-policy.adoc#ocp-versioning-policy[OpenShift
 
 See link:https://kubernetes.io/docs/setup/release/version-skew-policy/[Kubernetes documentation] for details about version skew policies in the upstream Kubernetes project.
 
+[discrete]
+[id="ocp-4-7-pod-topologys-spread-constraints"]
+==== Scheduler now uses pod topology spread constraints
```

Comment:
"Pod topology spread constraints" is a lot. Could it benefit from a hyphen? 

I know that this phrase doesn't just appear here, so, if you agree, that might just be something to stick a pin in.

Reply:
Yeah thanks for the question, but it comes from an upstream feature that doesn't use hyphens, so I think we'll leave it as is to be consistent.